libindy/src/services/anoncreds/prover.rs

```diff
@@ -413,13 +436,23 @@ impl Prover {
 
     pub fn extend_proof_request_restrictions(&self,
                                              version: &ProofRequestsVersion,
-                                             name: &str,
+                                             name: &Option<String>,
+                                             names: &Option<Vec<String>>,
```

Comment:
Are allocated string and vector required here and match the code on the level above?

Reply:
What do you mean by match the code?

Reply:
usability on the level above